src/main/drivers/bus_i2c.h

```diff
@@ -64,9 +80,20 @@ typedef struct i2cState_s {
     volatile uint8_t* read_p;
 } i2cState_t;
 
+typedef struct i2cPinConfig_s {
+    ioTag_t ioTagSCL[I2CDEV_MAX];
```

Comment:
I'll just leave it as is for now.

IMO, mcu functions with strong constraints about pins should be specified by something beyond what current resource command does, but not sure about the correct way (yet).